README.md

```diff
@@ -69,6 +69,8 @@ Don't forget to pass a `ref` to the component.
 
 You can change the initial options passing props to the component.
 
+**autoResizeHole** {bool}: Enable autoresizing holes when the content under the hole change its size. Defaults to `false`
```

Comment:
Why did you choose to make this optional?  It seems like it could pretty safely be always turned on, as I can't think of any use cases for wanting the hole to stay the same size while the element underneath changes size.